ot/mpz-ot/src/lib.rs

```diff
@@ -68,6 +68,30 @@ where
     ) -> Result<(), OTError>;
 }
 
+/// A correlated oblivious transfer sender.
+#[async_trait]
+pub trait COTSender<T>: ProtocolMessage
+where
+    T: Send + Sync,
+{
+    /// Obliviously transfers the correlations to the receiver.
```

Comment:
I changed "correlation" to "correlated messages" for consistency and clarity.
```suggestion
    /// Obliviously transfers the correlated messages to the receiver.
```